### PR TITLE
Merchant invoice show

### LINKED ISSUE #26: Merchant Invoice Show Page
As a merchant
When I visit my merchant's invoice show page(/merchants/merchant_id/invoices/invoice_id)
Then I see information related to that invoice including:
- Invoice id
- Invoice status
- Invoice created_at date in the format "Monday, July 18, 2019"
- Customer first and last name

### LINKED ISSUE #27: Merchant Invoices Index
As a merchant,
When I visit my merchant's invoices index (/merchants/merchant_id/invoices)
Then I see all of the invoices that include at least one of my merchant's items
And for each invoice I see its id
And each id links to the merchant invoice show page

### PR DESCRIPTION
**Pull request by JoePeterson51**
### Features being added/updated
Merchant Invoice Show Page

### Any issues this PR will unblock/block (Leave blank if no blockers are involved)

- [x] #57 has been merged
- [X] Ready for merge (Don't merge unless the author of the PR has checked this)
- [X] All Tests are passing

(Link issue number next to `#` to automatically move the card on kanban board)

closes #26 
closes #27

### CODE REVIEW

**Comment by redferret** [comment]
Let's wait until we merge #57 then merge this

**Review by redferret — APPROVED** [review]

**Review by JoannaCoPo — APPROVED** [review]
Looks good

**Pull request merged**
